### PR TITLE
feat: persit sort on filters

### PR DESCRIPTION
**Pull request by pirhoo**
This feature move the parameters to sort a filter to the store. This allow us to share the state between different instances of the Filter component and also to persist this value in the local storage.

### CODE REVIEW

File: src/store/modules/search.js [hunk 1]
```diff
@@ -46,8 +46,9 @@ export function initialState() {
     layout: isNarrowScreen() ? 'table' : 'list',
     query: '',
     response: EsDocList.none(),
-    reversed: [],
-    contextualized: [],
+    reversedFilters: [],
```
Comment (caro3801) [inline src/store/modules/search.js:49]: Nice renaming

**Review by caro3801 — APPROVED** [review]
:+1:

File: src/components/filter/FilterFooter.vue [hunk 2]
```diff
@@ -1,13 +1,6 @@
...
   <div v-if="!hideFooter" class="filter__footer d-flex align-items-center text-nowrap p-1">
-    <filter-sort-by-dropdown
-      v-if="!hideSort"
-      :sort-by="sortBy"
-      :sort-by-order="sortByOrder"
-      :sort-by-options="sortByOptions"
-      @update:sortBy="$emit('update:sortBy', $event)"
-      @update:sortByOrder="$emit('update:sortByOrder', $event)"
-    />
+    <filter-sort-by-dropdown v-if="!hideSort" v-model="sort" :sort-by-options="sortByOptions" />
```
Comment (marieglr) [inline src/components/filter/FilterFooter.vue:3]: fake review

File: src/components/filter/FilterFooter.vue [hunk 3]
```diff
@@ -110,6 +95,27 @@ export default {
...
+      get() {
+        const sortBy = this.sortBy
+        const sortByOrder = this.sortByOrder
+        return { sortBy, sortByOrder }
+      },
+      set({ sortBy, sortByOrder }) {
+        const name = this.filter.name
+        this.$store.commit('search/sortFilter', { name, sortBy, sortByOrder })
+        /**
+         * Triggered when the filter is "sorted"
```
Comment (marieglr) [inline src/components/filter/FilterFooter.vue:109]: aother

**Review by marieglr — COMMENTED** [review]
Thanks

**Pull request merged**